app/src/main/res/layout/fragment_upload_media_detail_fragment.xml

```diff
@@ -127,8 +127,8 @@
                       android:layout_width="wrap_content"
                       android:layout_height="wrap_content"
                       android:layout_alignParentRight="true"
-                      android:enabled="false"
                       android:alpha=".5"
+                      android:enabled="false"
```

Comment:
Is this change needed?

Reply:
No. I think this change is auto generated. I will revert it.